include/SFML/Graphics/RenderWindow.h

```diff
@@ -52,7 +52,7 @@
 /// \param settings Creation settings (pass NULL to use default values)
 ///
 ////////////////////////////////////////////////////////////
-CSFML_GRAPHICS_API sfRenderWindow* sfRenderWindow_create(sfVideoMode mode, const char* title, uint32_t style, const sfContextSettings* settings);
+CSFML_GRAPHICS_API sfRenderWindow* sfRenderWindow_create(sfVideoMode mode, const char* title, uint32_t style, uint32_t state, const sfContextSettings* settings);
```

Comment:
Can we use the `state` enum type here (and elsewhere were states are used) instead of `uint32_t`?

Reply:
Should be done.